### ISSUE TITLE
Emoji position overlap

### ISSUE THREAD

**Issue by go-gou** [serebrov/emoji-mart-vue#50]
Hi! Thank you for the good improvement!

Issue is about emoji overlap like this image. (I use emoji picker tag in popup dialog)
When I try original emoji-mart-vue,  no overlap shown.
Anyway please look at below screenshot.

**State 1: emoji overlap**
![Screenshot from 2019-07-12 03-21-51](https://user-images.githubusercontent.com/47044183/61078527-c2482d00-a45b-11e9-8239-03cd67782a04.png)

**State 1: CSS**
![Screenshot from 2019-07-12 04-24-12](https://user-images.githubusercontent.com/47044183/61079017-f112d300-a45c-11e9-867f-3ff1d38c284a.png)

**State 2**
After state1,  I **disabled** the check box about emoji-mart line-height(Chrome Dev Tool).
Then overlap  has been fixed. (State2)

![Screenshot from 2019-07-12 04-27-59](https://user-images.githubusercontent.com/47044183/61079287-78f8dd00-a45d-11e9-8854-edce8a07523b.png)

**State 2: CSS**
![Screenshot from 2019-07-12 04-43-43](https://user-images.githubusercontent.com/47044183/61080279-ad6d9880-a45f-11e9-871a-328db4351f62.png)

**State 3**
And after state 2,  I **enabled** the check box about emoji-mart line-height.
Then overlap **NOT** shown. Why ? (State3)
The State 1 and State 3 is same about css.
But something different ? It's cause emoji overlap ?
Or this issue related ? [(Only last item visible)](https://github.com/Akryum/vue-virtual-scroller/issues/96)

![Screenshot from 2019-07-12 04-27-59](https://user-images.githubusercontent.com/47044183/61079957-f2dd9600-a45e-11e9-9dac-b8ea74264044.png)

**State 3: CSS**
![Screenshot from 2019-07-12 04-44-14](https://user-images.githubusercontent.com/47044183/61080296-ba8a8780-a45f-11e9-8a0b-1b8820379d08.png)

**Comment by serebrov** [comment]
@go-gou can you check if it works for you in the [demo app](https://serebrov.github.io/emoji-mart-vue/)? 

The issue is probably caused either by some custom CSS or component configuration - can you share the code with the component configuration (or a small demo app to demonstrate the issue)?

As a quick fix, you can overwrite the standard CSS (more specific rules, or !important) if it doesn't work for you case and set the line-height that works. Another alternative is to not include the standard CSS, copy and modify it as needed.

**Comment by go-gou** [comment]
@serebrov thank you your comment.

I found the cause and the solution.

It was related to using the picker in the dialog.
I am using the dialog of the quasar framework.([q-dialog](https://quasar.dev/vue-components/dialog))

The default animation for the quasar dialog was scaling.
I changed this behavior to fade, then the overlap was resolved.

Postscript
Specifying "line-height: 1.5! important" in css did not resolve the overlap. Unfortunately, I could not figure out the cause in detail, but I solved this problem by not using scaling animation.
Thank you for your help.
